apps/src/applab/dataBrowser/ColumnHeader.jsx

```diff
@@ -149,6 +154,21 @@ const ColumnHeader = React.createClass({
               Delete
             </a>
           </li>
+          <li style={{cursor: 'pointer'}}>
+            <a onClick={() => this.coerceColumn('string')}>
+              Convert to string
+            </a>
+          </li>
+          <li style={{cursor: 'pointer'}}>
+            <a onClick={() => this.coerceColumn('number')}>
+              Convert to number
+            </a>
+          </li>
+          <li style={{cursor: 'pointer'}}>
+            <a onClick={() => this.coerceColumn('boolean')}>
+              Convert to boolean
+            </a>
+          </li>
```

Comment:
By having these functions defined in the onClick, we'll create a new function (or a bunch of new functions) every time we re-render. It might be more performant to instead call a method on the class (i.e. `onClick={this.coerceColumnString}`) tho it's also quite likely that this is an unnecessary performance optimization.